utility/mm2_generate-worldmap

```diff
@@ -80,6 +80,7 @@ def lookup_host_locations(config):
 
 def doit(options, config):
 
+    gi = geoip2.database.Reader('/usr/share/GeoIP/GeoLite2-City.mmdb')
```

Comment:
Wondering if that will always be in the same location

Reply:
Right. I was wondering the same. But as we used to hard-code the location to the old location I thought that we just keep doing it. If we ever need to change it, it will be minimal effort to extend to be configurable or command-line parameter. (and I am pressing the wrong buttons :-)